src/main/java/org/coffeemine/app/spring/components/HoursLogIn.java

```diff
@@ -0,0 +1,58 @@
+package org.coffeemine.app.spring.components;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.button.ButtonVariant;
+import com.vaadin.flow.component.datepicker.DatePicker;
+import com.vaadin.flow.component.dialog.Dialog;
+import com.vaadin.flow.component.formlayout.FormLayout;
+import com.vaadin.flow.component.notification.Notification;
+import com.vaadin.flow.component.textfield.TextArea;
+import com.vaadin.flow.component.textfield.TextField;
+import org.coffeemine.app.spring.data.Task;
+import java.time.LocalDate;
+
+public class HoursLogIn {
+    private Task newTask;
+    private Dialog dialog;
+    private FormLayout newForm;
+    private Button hoursLogIn;
+    private Button create;
+    private Button cancel;
+
+    public HoursLogIn(){
+        newTask = new Task();
+        dialog = new Dialog();
+        newForm = new FormLayout();
+    }
+
+    public void taskCreating(){
```

Comment:
why do i feel like you you straight up copped chens' old work?

Reply:
Chen showed his code to me on Friday so I did the same structure as his. But yes I should rename next time.